packages/first-boot-wizard/tests/test_firstbootwizard.lua

```diff
@@ -178,17 +178,46 @@ describe('FirstBootWizard tests #fbw', function()
 
         -- Assert reading data
         fbw.start_scan_file() -- simulate is scanning to don't start the scan
-        local results = fbw.start_search_networks()
+        local results = fbw.status_search_networks()
         assert(true, type(results['scanned']) == table)
         assert(true, utils.deepcompare(scanlist, results['scanned']))
     end)
 
-    it('test stop_get_all_networks', function()
+    it('test start search networks', function()
+        fbw.end_scan()
+        local results = fbw.start_search_networks()
+        assert(true, results)
```

Comment:
There are two issues here:
- The name of the test "test start search networks", should say what is actually being tested. Example: an script is executed, another function is called, it returns some expected value, etc.
- We are not testing anything with: `assert(true, results)` 
From [here](https://www.lua.org/pil/8.3.html) 
> The assert function checks whether its first argument is not false and simply returns that argument; if the argument is false (that is, false or nil), assert raises an error. Its second argument, the message, is optional, so that if you do not want to say anything in the error message, you do not have to.

You may want to use assert.is_true, assert.are_same, and such from https://olivinelabs.com/busted/#asserts

Reply:
This comments apply for the other tests aswell

Reply:
I rewrite all tests again reformating the code and it show me some errors on tests. Fixed after 4f7394ef